tests/run.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
```

Comment:
Having tests/run.sh only run the integration tests and not the unit tests is confusing, I think we should stick all integration-test related stuff in tests/integration to avoid this.